app/helpers/application_helper/button/vm_reset.rb

```diff
@@ -0,0 +1,13 @@
+class ApplicationHelper::Button::VmReset < ApplicationHelper::Button::Basic
+  needs_record
+
+  def skip?
+    !@record.is_available?(:reset)
+  end
+
+  def disable?
+    if @record.is_available_now_error_message(:reset)
+      return @record.is_available_now_error_message(:reset)
```

Comment:
I stumbled upon this in another PR. Why does `disable?` sometimes return a boolean and sometimes a message string?
Is the string really used somewhere?